data/sheragi/archaeology missions.txt

```diff
@@ -1289,42 +1289,104 @@ mission "Sheragi Archaeology: The Emerald Sword 3"
+			`	While the crew starts to work on the Emerald Sword, you and the archaeologists head into the settlement, documenting what they see. The buildings were all connected to each other to avoid the exterior atmosphere, although nowadays, not much remains. They were made of glass, scrap, and a kind of yellowish white plastic, the same plastic that the carapaces of animals here are made of. They were melted into shape and built into walls over time, giving the buildings a rounded, anachronistic look, like mud huts made of modern materials. All around the inside there are curtains and rags hanging from the ceiling to the ground. Closer inspection reveal those rags to have traces of chemicals, likely made to react with the chlorine in the air to keep the atmosphere inside breathable.`
+			`	The smaller rooms you come across appear to be just family homes, with a simplistic, single room interior, while the larger ones had other purposes, like what appears to have been a classroom, an animal carapace storage room, holding the exoskeletons of many different species, a communal kitchen with remains of a solar oven, and most strikingly, a hatchery, with relatively small beds, nests, and makeshift toys strewn about. Most of the rooms are littered with skeletons and remains, from the largest hunter-gatherers to the smallest young ones, a grim scene suggesting that some unknown catastrophe ended their lives all at once. Even then, the Sheragi on the surface apparently survived for much longer than the ones in the Emerald Sword, perhaps for many generations. Had the ship landed, they could have survived to this day. The archaeologists take pictures, samples, and minor artifacts, but leave the place mostly undisturbed in respect for a lost civilization, discussing the possible history of this place.`			
+			`	For most of the next day, the workers unload some outfitting and repairing equipment, while the spare crew take turns thinking of names for each fighter, and scientists slowly work out the functions of the ship, briefing you and your crew on how to operate it. With the Jump Drive installed, all that is left is to return the ship to human space for further work.`
 
+mission "Sheragi Archaeology: give ship 2"
+	landing
+	invisible
+	source "Echoes of Tragedy"
+	destination "Zug"
+	to offer
+		has "Sheragi Archaeology: The Emerald Sword 3: done"
+	on offer
+		give ship "Demonheart" "Discount Flak Round"
```

Comment:
Umm... you can give multiple ships in a single mission.

Reply:
I should've made a bug report about this.
The give ship command apparently doesn't work to give multiple ships of the same type.
When i attempted to give all 7 ships on the same mission, only the emerald sword, and the last fighter were given.
When i attempted to give the emerald sword and a fighter on the first mission, and more fighters on different missions, the first fighter that was supposed to be given along with the emerald sword wasn't given.
This was the workaround i ended up with, but there is something wrong with the code.

Reply:
I've now submitted a detailed bug report on #5447